jnigen/test/yaml_config_test.dart

```diff
@@ -1,30 +0,0 @@
-// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
```

Comment:
Isn't this test testing anything over the other tests that we remove it? (Apparently we don't drop coverage...)

Reply:
I guess now that we use `package:cli_config`, we can rely on the tests there.

Reply:
> Isn't this test testing anything over the other tests that we remove it? (Apparently we don't drop coverage...)

Apparently because we have a test which compares the same YAML with programmatic config line by line, and the latter is used in integration test!